x-pack/plugins/event_log/generated/mappings.json

```diff
@@ -292,6 +292,12 @@
                                                 "number_of_triggered_actions": {
                                                     "type": "long"
                                                 },
```

Comment:
Saw this was just added in https://github.com/elastic/kibana/pull/123567, shall we add it to the table as well, or is it getting too congested and better to wait till we allow expanding rows with additional data?